src/tools/adig.c

```diff
@@ -152,6 +155,7 @@ static const struct nv types[] = {
   { "DNSKEY",   T_DNSKEY },
   { "CAA",      T_CAA },
   { "ANY",      T_ANY }
+  { "URI",      T_URI }
```

Comment:
Add a `,` above that line. You probably have noted the AppVeyor jobs have failed.